fiona/ogrext1.pyx

```diff
@@ -151,6 +153,8 @@ cdef class FeatureBuilder:
         cdef int mm = 0
         cdef int ss = 0
         cdef int tz = 0
+        cdef unsigned char *data
+        cdef int l
```

Comment:
This will need casting to an `int` later, as that's what `OGR_F_GetFieldAsBinary` expects. Is that OK?